build.gradle

```diff
@@ -37,9 +37,9 @@ configure(subprojects) {
     }
 
     repositories {
-        mavenCentral()
```

Comment:
Please don't change this. There are issues with JCenter's SSL cert under OpenJDK 10, that's why I recently switched everything to Maven Central.